exporter/stackdriverexporter/stackdriver.go

```diff
@@ -30,10 +30,13 @@ import (
 )
 
 type stackdriverConfig struct {
-	ProjectID     string `mapstructure:"project,omitempty"`
-	EnableTracing bool   `mapstructure:"enable_tracing,omitempty"`
-	EnableMetrics bool   `mapstructure:"enable_metrics,omitempty"`
-	MetricPrefix  string `mapstructure:"metric_prefix,omitempty"`
+	ProjectID       string                 `mapstructure:"project,omitempty"`
+	EnableTracing   bool                   `mapstructure:"enable_tracing,omitempty"`
+	EnableMetrics   bool                   `mapstructure:"enable_metrics,omitempty"`
+	MetricPrefix    string                 `mapstructure:"metric_prefix,omitempty"`
+	TraceAttributes map[string]interface{} `mapstructure:"trace_attributes,omitempty"`
```

Comment:
Any reason to not use [global tags](https://github.com/Omnition/internal-opencensus-service#-global-tags) instead? In general we've been approaching this type of issue making it happen in the oc proto format so all exporters can generate the same data.

We still don't have the same for metric labels and prefix but both are applicable too.

Reply:
That URL 404's

Reply:
ops, just fixed the link

Reply:
If you have links for other docs, that'd be great.  My searching came up empty.  If I can use an existing feature, I'd rather do so.  Thanks!

Reply:
Right now we just have global tags for tracing https://github.com/census-instrumentation/opencensus-service#-global-tags but we should implement the equivalent for metrics /cc @songy23

Reply:
btw, there is an issue regarding naming: on OC we should refer to these as attributes instead of tags but the docs were not updated yet.

Reply:
another related feature for attributes is in the pipelines https://github.com/census-instrumentation/opencensus-service/pull/536

Reply:
Hi @bakins, given that #536 was merged today, do you still need to add some functionality from this PR?